feature/schedule/src/main/java/com/yapp/feature/schedule/ScheduleViewModel.kt

```diff
@@ -29,10 +29,19 @@ class ScheduleViewModel @Inject constructor(
         postSideEffect: (ScheduleSideEffect) -> Unit,
     ) {
         when (intent) {
-            ScheduleIntent.EnterScheduleScreen -> loadScheduleInfo(state.selectedYear, state.selectedMonth, reduce)
+            ScheduleIntent.EnterScheduleScreen -> {
+                loadScheduleInfo(state.selectedYear, state.selectedMonth, reduce)
+            }
             is ScheduleIntent.SelectTab -> {
                 if (state.selectedTab != intent.tab) {
                     reduce { copy(selectedTab = intent.tab) }
+                    when (intent.tab) {
+                        ScheduleTab.ALL -> loadScheduleInfo(state.selectedYear, state.selectedMonth, reduce)
+                        ScheduleTab.SESSION -> {
+                            loadUpcomingSessionInfo(reduce)
+                            loadSessions(reduce)
+                        }
+                    }
                 }
```

Comment:
```suggestion
                if (state.selectedTab != intent.tab) {
                   return
                }
                    reduce { copy(selectedTab = intent.tab) }
                    when (intent.tab) {
                        ScheduleTab.ALL -> loadScheduleInfo(state.selectedYear, state.selectedMonth, reduce)
                        ScheduleTab.SESSION -> {
                            loadUpcomingSessionInfo(reduce)
                            loadSessions(reduce)
                        }
                    }
```

A: 얼리 리턴을 사용하면 조금 더 가독성이 좋아질거같아요!

Reply:
넵 ㅈ거용했습니다.